wgpu-types/src/lib.rs

```diff
@@ -2076,3 +2563,18 @@ pub struct TextureCopyView<T> {
     /// The base texel of the texture in the selected `mip_level`.
     pub origin: Origin3d,
 }
+
+impl<T> TextureCopyView<T> {
+    pub fn new(texture: T, origin: Origin3d) -> Self {
```

Comment:
let's remove the origin from here (defaulting to `ZERO`), and have a method for it